pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.25</version>
+    <version>2.37</version>
```

Comment:
The latest is 3.x, but I heard that 3.x officially supports only core 2.x.
https://github.com/jenkinsci/copyartifact-plugin/pull/97#discussion_r155136724